tx-pool/src/process/submit_txs.rs

```diff
@@ -242,6 +242,11 @@ fn resolve_tx<'a>(
     }
 
     let short_id = tx.proposal_short_id();
+
+    if tx_pool.contains_proposal_id(&short_id) {
```

Comment:
Can you move this condition to `process_txs`.

Reply:
I move it in PreResolveTxsProcess